core/src/components/refresher/test/test.utils.ts

```diff
@@ -12,10 +11,28 @@ import { dragElementBy } from '@utils/test';
  * @param selector The element selector to center the drag gesture on. Defaults to `body`.
  */
 const pullToRefresh = async (page: E2EPage, selector = 'body') => {
-  const target = (await page.$(selector))!;
+  const target = page.locator(selector);
 
-  await dragElementBy(target, page, 0, 200);
-  const ev = await page.spyOnEvent('ionRefreshComplete', 'document');
+  await page.waitForSelector('ion-refresher.refresher-inactive', { state: 'attached' });
```

Comment:
Rather than wait for a new class, are you able to do `expect(refresher).not.toHaveClass(/refresher-active/)`?

Reply:
I believe this would operate differently than the current behavior. Using an `expect` check here will continue to fail on the few test suites that's threshold of navigating to the test and starting the mouse down behavior, happens prior to the component fully rendering and the gestures being initialized. 

Optionally, I could swap this behavior out with:
```ts
 await page.waitForSelector('ion-refresher.hydrated', { state: 'attached' });
```

since Playwright runs on the lazy bundle. I can swap that out quick and make sure that CI still passes.

Reply:
Looks like this approach works. Are you comfortable with this change?

Reply:
yep works for me